### DISCUSSION TITLE
Welcome to Rox.Control.Camera.Xamarin Discussions

### DISCUSSION THREAD

**Discussion by ghost** [Rod-at-Rox/RoxXamarinCamera#11] [Announcements]
## 👋 Welcome to Rox Camera Control for Xamarin.Forms
We’re using Discussions as a place to connect with other members of our community. We hope that you:
* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

Feel free to comment below and tell us about what you do with this community.

**Comment by r-o-d-p** [comment]
Hello! My name is Rod and I am working on development tasks as they arise.
